cellrank/tl/kernels/_pseudotime_schemes.py

```diff
@@ -164,6 +165,12 @@ def __call__(
         -------
         %(pt_scheme.returns)s
         """
+        if frac_to_keep < 0 or frac_to_keep > 1:
```

Comment:
Change it to this:
```python
if not (0 <= frac_to_keep <= 1):
    raise ValueError(f"Expected `frac_to_keep` to be in `[0, 1]`, found `{frac_to_keep}`.`")
```
I usually write error messages in this format (i.e. `Expected ..., found ...`).